servers/quarkus-server/src/test/java/org/projectnessie/server/TestRest.java

```diff
@@ -546,4 +551,9 @@ void unwrap(Executable exec) throws Throwable {
       throw targetException;
     }
   }
+
+  private static void assertEqualsContentsKey(ContentsWithKey expected, ContentsWithKey actual) {
```

Comment:
Super nit: maybe assertContentWithKeyEquals (sounds "better" in my German ears)

Reply:
fixed